packages/lab/src/carousel/CarouselControls.tsx

```diff
@@ -0,0 +1,122 @@
+import { Button, Text, makePrefixer, useIcon } from "@salt-ds/core";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+import {
+  type HTMLAttributes,
+  type MouseEvent,
+  type SyntheticEvent,
+  forwardRef,
+  useRef,
+} from "react";
+import { useCarousel } from "./CarouselContext";
+
+import carouselControlsCss from "./CarouselControls.css";
+
+const withBaseName = makePrefixer("saltCarouselControls");
+
+export interface CarouselControlsProps
+  extends Omit<HTMLAttributes<HTMLDivElement>, "title"> {
+  /**
+   * Callback when back button is clicked.
+   */
+  onMoveBack?: (event: SyntheticEvent<HTMLButtonElement>) => void;
```

Comment:
we changed the language on this api, should it be onNext and onPrevious, or similar ?
this will align with the rest of the code